docs/guide.md

```diff
@@ -149,6 +172,16 @@ Notable examples of mods which will **not work properly** without Indium install
 - Materialisation
 - Simple BC Pipes
 
+... and many others which could possibly utilize the Fabric Rendering API, **you will experience issues when playing the game**.
+
+**Solution**: Download **Indium** ([Curseforge](https://www.curseforge.com/minecraft/mc-mods/indium), [Modrinth](https://modrinth.com/mod/indium)) and add it to your mods folder.  
+
+Make sure to download the version that supports your current version of Sodium:
+
+| Sodium       | 1.16.5-0.2.0        | 1.17.1-0.3.4        | 1.18.1-0.4.0-alpha6 |
```

Comment:
This table should probably be transposed too.

Reply:
Done